### PR TITLE
fixes for latest WME

### PR DESCRIPTION
**Pull request by dalverson**

### CODE REVIEW

File: src/include/recurringClosures.js [hunk 1]
```diff
@@ -232,7 +232,7 @@ WMEAC.refreshClosureListFromSelection = function ()
         {
             var blackList=[];
             W.model.roadClosures.getObjectArray(function (c) {
-                return c.segID==W.selectionManager.getSelectedFeatures()[0].model.attributes.id;
+                return c.segID==W.selectionManager.getSelectedFeatures()[0].attributes.wazeFeature.id;
```
Comment (justins83) [inline src/include/recurringClosures.js]: I would change this to W.selectionManager.getSelectedDataModelObjects()[0].id

File: src/include/setupUI.js [hunk 2]
```diff
@@ -536,13 +537,16 @@ WMEAC.connectAdvancedClosureDialogHandlers = function ()
...
                 return;
             }
-            if (W.selectionManager.getSelectedFeatures().length==0 || W.selectionManager.getSelectedFeatures()[0].model.type!="segment")
+            const m = W.selectionManager.getSelectedDataModelObjects();
+            if (m.length==0 || m[0].type != "segment")
             {
                 alert("Please, select segment(s) before.");
                 return;
             }
             if (W.selectionManager.getSelectedFeatures().every(function (e) {
```
Comment (justins83) [inline src/include/setupUI.js]: I would modify this to W.selectionManager.getSelectedDataModelObjects().every...
and
const segid = e.id;

File: src/include/recurringClosures.js (hunk 1 above)
Comment (dalverson) [reply src/include/recurringClosures.js]: Looks simpler. Thanks.

File: src/include/setupUI.js (hunk 2 above)
Comment (dalverson) [reply src/include/setupUI.js]: Looks good. I will make those changes.

Comment (dalverson) [reply src/include/setupUI.js]: For some reason, e.id is null here. e.attributes.id would work.

File: src/include/recurringClosures.js (hunk 1 above)
Comment (dalverson) [reply src/include/recurringClosures.js]: This will be the same problem. The id at the root of that returned data is null.

Comment (dalverson) [reply src/include/recurringClosures.js]: changed to use getSelectedDataModelObjects()[]  .atttributes.id here

File: src/include/setupUI.js (hunk 2 above)
Comment (dalverson) [reply src/include/setupUI.js]: changed to use getSelectedDataModelObjects()[]  .atttributes.id here

**Pull request merged**